### PR TITLE
[voice] make getVoice public

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
This will for example allow checking the voice parameter in the new REST voice API before calling say.

File: bundles/org.openhab.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceManagerImpl.java [hunk 1]
```diff
@@ -280,7 +280,7 @@ public String interpret(String text, String hliId) throws InterpretationExceptio
         return interpreter.interpret(localeProvider.getLocale(), text);
     }
 
-    private Voice getVoice(String id) {
+    public Voice getVoice(String id) {
```
Comment (kaikreuzer) [inline bundles/org.openhab.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceManagerImpl.java]: This is an internal method that is not part of the interface. As nobody should depend on the impl class, it imho makes no sense to make this public here.

Comment (lolodomo) [reply bundles/org.openhab.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceManagerImpl.java]: Good point. 
So this is the interface that has to be enhanced.

**Comment by lolodomo** [comment]
I close this PR.

**Pull request closed without merging**